packages/store/src/text-adapter.ts

```diff
@@ -231,38 +231,43 @@ export class Text {
   }
 
   sliceToDelta(begin: number, end?: number) {
+    const result: DeltaOperation[] = [];
     if (end && begin >= end) {
-      return [];
+      return result;
     }
 
     const delta = this.toDelta();
     if (begin < 1 && !end) {
       return delta;
     }
-    const result = [];
+
     if (delta && delta instanceof Array) {
       let charNum = 0;
       for (let i = 0; i < delta.length; i++) {
-        const content = delta[i];
-        let contentText = content.insert || '';
+        const content: DeltaOperation = delta[i];
```

Comment:
```suggestion
        const content = delta[i];
```

Reply:
I think using `any` everywhere is not a good idea. 🤔

Reply:
sorry if misleading, I mean make `toDelta` to

```ts
toDelta(): DeltaOperation[] {
```